packages/dnb-eufemia/src/components/table/Table.tsx

```diff
@@ -20,8 +20,8 @@ import {
   StickyTableHeaderProps,
 } from './TableStickyHeader'
 
-export type TableSizes = 'medium' | 'large'
-export type TableVariants = 'basis' | 'not-defined-yet'
+export type TableSizes = 'large' | 'medium'
+export type TableVariants = 'generic'
```

Comment:
```suggestion
export type TableVariants = basis' | 'generic'
```

To me it seems like this is not matching the sentence in [properties docs](https://github.com/dnbexperience/eufemia/pull/1699/files#diff-fdc0e616e17cbb5f8abe78c2285bfacb789c29e4e5323bd3de0aedd9199d66aaR11), _Options: basis . Defaults to generic_. It's fine though, I'm probably just missing some of the "context" 😄